backend/services/API.js

```diff
@@ -8,13 +8,21 @@ const reqTimeout = 1000 * 5;
  * specific amount of data according to searchTerm.
  */
 async function getSearchedImages (apiType, limit, searchTerm, offset) {
```

Comment:
I think this function is getting complicated and harder to read, considering we will be adding more APIs. We should split this per API and use it within its routes folder or somewhere else. What do you think ?

Reply:
yes, I think this will be good.
We can make different functions for search and trendy images in the respective file of different APIs which are inside the routes folder.
Should I make the required changes and delete this services folder because after we implement the functions in their respective file, we no longer need this folder?